CHANGELOG.md

```diff
@@ -1,48 +1,59 @@
-## 0.11.0 (2018-10-08)
+### 0.12.0 / 2019-10-08
```

Comment:
Should the date be in parentheses like the other dates in this changelog?

Reply:
Or should the parentheses be removed from the other dates to match the new format?

Reply:
Okay updated the other dates to match this format. (This format is the one generated by releasetool.)